ShchyrbaOleksandr/HW10/task1.py

```diff
@@ -0,0 +1,19 @@
+class polygon:
```

Comment:
it is better version of your code

class Polygon:
    def __init__(self):
        pass


class Rectangle(Polygon):
    def __init__(self):
        super().__init__()

    @staticmethod
    def enter_sides():
        side_a = int(input("Enter side a: "))
        side_b = int(input("Enter side b: "))
        return Rectangle.area_rectangle(side_a, side_b)

    @staticmethod
    def area_rectangle(side_a, side_b):
        return side_a * side_b

Rectangle.enter_sides()